src/lib/homestore_backend/tests/hs_blob_tests.cpp

```diff
@@ -155,39 +178,13 @@ TEST_F(HomeObjectFixture, BasicPutGetDelBlobWRestart) {
     restart();
 
     // Verify all get blobs after restart
-    for (const auto& [id, blob] : blob_map) {
-        int64_t pg_id = std::get< 0 >(id), shard_id = std::get< 1 >(id), blob_id = std::get< 2 >(id);
-        auto g = _obj_inst->blob_manager()->get(shard_id, blob_id).get();
-        ASSERT_TRUE(!!g);
-        auto result = std::move(g.value());
-        LOGINFO("After restart get blob pg {} shard {} blob {} data {}", pg_id, shard_id, blob_id,
-                hex_bytes(result.body.bytes, std::min(10u, result.body.size)));
-        EXPECT_EQ(result.body.size, blob.body.size);
-        EXPECT_EQ(std::memcmp(result.body.bytes, blob.body.bytes, result.body.size), 0);
-        EXPECT_EQ(blob.user_key, result.user_key);
-    }
+    verify_get_blob(blob_map);
 
-    // Verify all get blobs with random offset and length.
-    for (const auto& [id, blob] : blob_map) {
-        std::uniform_int_distribution< uint32_t > rand_off_gen{0u, blob.body.size - 1u};
-        std::uniform_int_distribution< uint32_t > rand_len_gen{1u, blob.body.size};
+    // Put blob after restart to test the persistance of blob sequence number
+    put_blob(blob_map, pg_shard_id_vec, num_blobs_per_shard, max_blob_size);
```

Comment:
how you verify this ? should we check the blob_id existence in blob_map?

Reply:
The index layer in blob manager will return an error if we try to insert a blob id that exists. If the blob id is not persisted, the counter is reset to 0 after restart and we will get the error. I ran this test without my persistence change and it was failing as expected.

Reply:
thx yes we have 
```
    homestore::BtreeSinglePutRequest put_req{&index_key, &index_value,
                                             homestore::btree_put_type::INSERT_ONLY_IF_NOT_EXISTS};
```